Turning on Module_IOTransformDCMTK results in CMake configuration error on windows

On Windows turning on `Module_IOTransformDCMTK` results in the following configuration error:
```
CMake Error at Modules/ThirdParty/DCMTK/CMakeLists.txt:379 (set_property): set_property could not find TARGET dcmdata.  Perhaps it has not yet been created.
```

--- a/Modules/ThirdParty/DCMTK/CMakeLists.txt
+++ b/Modules/ThirdParty/DCMTK/CMakeLists.txt
@@ -378,7 +378,7 @@ endforeach()
         string(TOUPPER "${c}" C)
         set_property(
           TARGET
-            ${lib}
+            ${ITK_MODULE_${itk-module}_TARGETS_NAMESPACE}${lib}
           PROPERTY
             IMPORTED_LOCATION_${C}
               ${lib_dir}/${c}/${lib_prefix}${lib}${lib_suffix}

